src/main/java/com/oitq2/kit/Kit.java

```diff
@@ -0,0 +1,42 @@
+package com.oitq2.kit;
+
+import org.bukkit.Material;
+import org.bukkit.entity.Player;
+import org.bukkit.inventory.EquipmentSlot;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.potion.PotionEffectType;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public final class Kit {
+
+    private List<String> abilities;
```

Comment:
This doesn't actually do anything, I would hold off on adding anything ability related for now, and focus on items, armor, potion effects, etc